docs/backup-restore.md

```diff
@@ -0,0 +1,106 @@
+# Scalar DL Backup Creation and Restoration
+
+Since Scalar DL provides transaction capability on top of non-transactional (possibly transactional) databases non-invasively, you need to take special care of backing up and restoring the databases in a transactionally-consistent way.
+This guide shows you how to create and restore transactionally-consistent Scalar DL backups.
+
+## How to create transactionally consistent backup 
+
+Scalar DL service is integrated with the admin interface, which allows you to pause the Scalar DL services using the scalar-admin client tool.
+You can use the scalar-admin client tool as a docker container or  fat jar, you can find scalar-admin-<version>-all.jar in the [releases](https://github.com/scalar-labs/scalar-admin/releases).
+
+### Requirements
+
+* You must wait at least 10 sec after pausing to create a backup.
+* You must identify a common restore point for the Ledger and Auditor if you use Auditor.
+### Pause
+
+The `PAUSE` command helps you to create a transactionally consistent backup for non-transactional databases. It helps to pause the Scalar DL service.
+
+```console
+java -jar scalar-admin-<version>-all.jar -c PAUSE -s <SRV_Service_URL>
+
+OR
```

Comment:
`OR` should be excluded from the console so that users would copy the command.

Reply:
Sorry for the inconvenience.
Document updated with new structure and sentences.